app/src/main/java/com/github/se/studybuddies/ui/groups/GroupScreen.kt

```diff
@@ -69,7 +70,8 @@ fun GroupScreen(
     groupUID: String,
     groupViewModel: GroupViewModel,
     chatViewModel: ChatViewModel,
-    navigationActions: NavigationActions
+    navigationActions: NavigationActions,
+    db: DbRepository
```

Comment:
Note: another common practice is to have default arguments for db manual injection, e.g.

```kotlin
db: DbRepository = DatabaseConnection.getInstance() // using the singleton design pattern
```

This way, you don't have to pass the db to every component like you did in this PR (=> more convenient)

Reply:
Thank you so much for your comment ! That's a small adjustment that can improve the code thanks !